Meter.c

```diff
@@ -51,32 +51,39 @@ Meter* Meter_new(const Machine* host, unsigned int param, const MeterClass* type
    return this;
 }
 
-int Meter_humanUnit(char* buffer, unsigned long int value, size_t size) {
-   const char* prefix = "KMGTPEZY";
-   unsigned long int powi = 1;
-   unsigned int powj = 1, precision = 2;
+/* Converts 'value' in kibibytes into a human readable string.
+   Example output strings: "0K", "1023K", "98.7M" and "1.23G" */
+int Meter_humanUnit(char* buffer, double value, size_t size) {
```

Comment:
What about `xSnprintf` usage here? Also I'd suggest adding braces for all `if` statements.